.gitignore

```diff
@@ -1,6 +1,4 @@
-.vs/
 build/
 sandbox/shaders/
-__pycache__
 *.pyc
-*.pyo
```

Comment:
Good question! `*.pyo` files were eliminated in Python 3.5, and in practice we usually don't have any such files even when using Python 2.7 because we never compile Python with optimizations (since it doesn't make much of a difference).

Reply:
If you do have `*.pyo` files in your working copy, please let me know and I'll reconsider that change. Thanks!